package/lib-mlpref_loadgen-static/install.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+
+#
+# CK installation script
+#
+# See CK LICENSE for licensing details.
+# See CK COPYRIGHT for copyright details.
+#
+
+# PACKAGE_DIR
+# INSTALL_DIR
+
+cd ${CK_ENV_MLPERF_INFERENCE_LOADGEN}
+echo "Loadgen source directory: ${CK_ENV_MLPERF_INFERENCE_LOADGEN}"
+cmake .
```

Comment:
We should pass the C++ compiler and flags here.

Reply:
For example, see https://github.com/ctuning/ck-armnn/blob/master/package/lib-armnn/scripts.linux/install.sh#L117